modules/amf-registration/amf-registration-rest/amf-registration-rest-impl/src/main/java/com/liferay/docs/modules/internal/resource/v1_0/AccountResourceImpl.java

```diff
@@ -13,4 +18,20 @@
 	scope = ServiceScope.PROTOTYPE, service = AccountResource.class
 )
 public class AccountResourceImpl extends BaseAccountResourceImpl {
+
+@Override
+public Account createAccount(Account account) throws Exception {
+	_accountLocalService.createAccount(account.getFirstName(), account.getLastName(), account.getEmailAddress(),
+			account.getUserName_(), account.getGenre(), account.getBirthday(), account.getPassword(),
+			account.getPassword2(), account.getHomePhone(), account.getMobilePhone(), account.getAddress1(),
+			account.getAddress2(), account.getCity(), account.getState(), account.getZip(),
+			account.getSecurityQuestion(), account.getSecurityAnswer(), account.getAcceptedTou());
+	return account;
```

Comment:
when implementing the logic here, I like to get the `Account` (model) returned from the `service` method, convert it to `Account` (dto) and then return it, just to make sure the inputs are being assigned and returned properly. I also use a `mapper` class with a `toDTO` method within it to make such conversion.